test/loading-serialization-test.coffee

```diff
@@ -205,10 +205,32 @@ describe "Serialization and Loading", ->
       @graphStore.nodeKeys['a'].image.should.equal "img/nodes/chicken.png"
       @graphStore.nodeKeys['b'].image.should.equal "img/nodes/egg.png"
 
+
     it "should load the nodes previous frames", ->
       data = JSON.parse(@serializedForm)
       data.nodes[1].frames = [50]
       @graphStore.loadData(data)
       @graphStore.nodeKeys['a'].frames.length.should.equal 0
       @graphStore.nodeKeys['b'].frames.length.should.equal 1
       @graphStore.nodeKeys['b'].frames[0].should.equal 50
+
+    it "Should load saved `combineMethod` for nodes if present", ->
+      json = '{
```

Comment:
Minor, perhaps call this `sampleNodes` instead of `json` to be a little more self-descriptive.